Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -51,26 +51,56 @@ Yes, CMILib is a library by Zrips that all his plugins require, including Jobs-R
 
 ### The plugin doesn't start!
 
-That sucks, but we require more info, does it say in the console (latest.log has this info as well) that the error message starts with 'missing dependency, CMILib required, download it' ? If so, the reason is that there's a missing dependency, the CMILib is required. You should download it.
+That sucks, but we require more info to help. Does the Jobs-related error message in your console say 'missing dependency, CMILib required, download it' ?
```

Comment:
Maybe say about where to find the console MSG's first, before, to keep the flow

IE:
That sucks that it isn't working, but we need more info...
In your console there will be a msg relating to jobs...
But don't worry if you missed it... Latest.log...

If the error says...